modelplugin/aether-1.0.0/aether.tree

```diff
@@ -0,0 +1,46 @@
+module: access-profile
+  +--rw access-profile
+     +--rw access-profile* [id]
+        +--rw id        access-profile-id
+        +--rw type?     string
+        +--rw filter?   string
```

Comment:
I think it would be good to have a description leaf in all of these profiles too - would make it a better experience on the GUI

Reply:
Will update in next patch.